README.md

```diff
@@ -0,0 +1,108 @@
+
+## Prerequisites
+
+Make sure you have the following software installed on your computer before proceeding:
+
+- Java Development Kit (JDK)
+- PostgreSQL database server
+- Eclipse IDE
+
+- About PostgreSQL Installation:
+During the installation you may get following error:
+
+"Problem running post-install step. Installation may not complete correctly
+The database cluster initialisation failed."
+
+To install PostgreSQL without any error, do the following:
+1. If you have a 3rd party Antivirus Software, pause it's protection temporarily.
+2. During installation, when it asks you for language, select "English - United States".
+
+The installation should be complete without any error.
+
+### Spring Tools 4 (Spring Tool Suite 4) 
+To ensure that you have Spring Tools 4 (Spring Tool Suite 4) installed in your Eclipse IDE, do the following:
+
+1. Open Eclipse IDE.
+2. Select the **Help** menu and click on **Eclipse Marketplace**.
+3. In the window opened, choose **Installed** tab to list solutions that are installed in your IDE.
+4. If there is **Spring Tools 4 (aka Spring Tool Suite 4) 4.x.x.RELEASE**, you have Spring Tools 4 installed in your IDE.
+If not, do following:
+	- Choose **Search** tab in Eclipse Marketplace window.
+	- Search for **Spring Tools 4 (aka Spring Tool Suite 4) 4.x.x RELEASE
+	- Follow the installation steps.
+
+## Steps
+
+### Importing the Project
+
+1. Open Eclipse IDE.
+2. Select the **File** menu and choose the **Import** option.
+3. In the opened window, select **Existing Maven Projects** and click **Next**.
+4. Specify the directory where the project is located (Root Directory) and click **Finish**.
+5. Eclipse will import the Maven project and add it to the project explorer.
+
+### Lombok Installation
+
+1. Ensure that Lombok is installed in your IDE.
+2. If Lombok is not installed, follow these steps:
+   - Close your IDE.
+   - Add the Lombok library to your IDE.
+   - Start your IDE.
+
+### Loading Dependencies and Building the Project
+
+After importing the project, Eclipse will automatically load the dependencies using Maven. This process may take some time and runs in the background. To compile your Maven project, follow these steps:
+
+1. Right-click on the project and select **Run As** from the context menu, then click on **Maven Install**.
+2. When the Maven project build process is complete, you will see a successful message in the console.
+
+### PostgreSQL Integration
+
+To use the PostgreSQL database server, follow these steps:
+
+1. Open the `application.properties` file located under `src/main/resources`.
+2. Update your database connection information according to your PostgreSQL server (e.g., URL, username, password, etc.).
+3. Save the changes.
+
+- Note that these database connection information is only for you. To keep changes for a specific file only in local and not track them
+without adding it into gitignore file, use following command on git bash:
```

Comment:
1. application.properties dosyasını gitignore'a eklememeliyiz. Cümle bu şekilde söylüyor. 
2. application.properties değil application-dev.properties dosyasının deişiklikleri takip edilmemeli ve assume-unchanged dev dosyası için kullanılmalı.

Reply:
Dosyanın orijinalinin repository'de durması gerektiğini düşündüğüm için gitignore'a eklemeden diye yazmıştım.

Reply:
ne yapmayacağımızı değil de ne yapacağımızı yazarsakdaha uygun olacaktır.

Reply:
Tamam Hocam. Ben okurken okuyucu "bunu gitignore'da da yapabilirim" diye düşünür diye öyle açıklama yapmıştım. Düzenliyorum şimdi.